x-pack/plugins/security_solution/public/common/components/sessions_viewer/index.tsx

```diff
@@ -54,6 +55,13 @@ export const defaultSessionsFilter: Required<Pick<Filter, 'meta' | 'query'>> = {
           },
         },
       ],
+      must_not: [
```

Comment:
potentially we should be able to join the check for existing and non-empty `entity_id`, smth like 
```
{
  "query": {
    "bool": {
      "filter": [
        {
          "terms": {
            "event.action": ["exec", "executed", "fork", "end"]
          }
        },
        {
          "bool": {
            "filter": [
              {
                "exists": {
                  "field": "entry_leader.entity_id"
                }
              },
              {
                "wildcard": {
                  "entry_leader.entity_id": "?*"
                }
              }
            ]
          }
        }
      ]
    }
  }
}

```
not a blocker, just allows to have the checks closer together logically

Reply:
Okay I will test this change and create a follow-up PR.